pkg/sdkserver/sdkserver.go

```diff
@@ -440,6 +445,9 @@ func (s *SDKServer) Shutdown(ctx context.Context, e *sdk.Empty) (*sdk.Empty, err
 	s.logger.Debug("Received Shutdown request, adding to queue")
```

Comment:
Can you update the function comment (line 438) to explain that shutdown will block on the game server being shutdown (if the graceful termination feature is enabled)?